### PR TITLE
fix(cli): add console command in the help message

### LINKED ISSUE #1607: amplify console undocumented in cli for a reason?
Running `amplify` or `amplify api` does not produce the `console` command in cli help. 

Is this intentional? I had only known of its existence by stumbling upon it in the docs somewhere.

### PR DESCRIPTION
**Pull request by UnleashedMind**
fix #1607

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by UnleashedMind** [comment]
combed through all the help messages to make them accurate.

File: packages/amplify-category-function/commands/function.js [hunk 1]
```diff
@@ -38,6 +38,10 @@ module.exports = {
         name: 'invoke <resourcename>',
         description: 'Invokes a function locally for testing',
       },
+      {
+        name: 'console',
```
Comment (kaustavghosh06) [inline packages/amplify-category-function/commands/function.js]: I don't think we support console for this category yet....

File: packages/amplify-category-storage/commands/storage.js [hunk 2]
```diff
@@ -30,6 +30,10 @@ module.exports = {
         name: 'remove',
         description: `Removes ${featureName} resource from your local backend. The resource is removed from the cloud on the next push command.`,
       },
+      {
```
Comment (kaustavghosh06) [inline packages/amplify-category-storage/commands/storage.js]: I don't think we support console for this category yet....

Comment (UnleashedMind) [reply packages/amplify-category-storage/commands/storage.js]: there's is such a command though, but it prints out to be implemented. 

    context.print.info(`to be implemented: ${category} ${subcommand}`);

File: packages/amplify-category-analytics/commands/analytics.js [hunk 3]
```diff
@@ -26,6 +26,10 @@ module.exports = {
         name: 'remove',
         description: `Removes ${featureName} resource from your local backend. The resource is removed from the cloud on the next push command.`,
       },
+      {
+        name: 'console',
+        description: `Opens the web console for the  ${featureName} category`,
```
Comment (SwaySway) [inline packages/amplify-category-analytics/commands/analytics.js]: nit: two spaces between the `${featureName}`

File: packages/amplify-category-auth/commands/auth.js [hunk 4]
```diff
@@ -30,6 +30,10 @@ module.exports = {
         name: 'update',
         description: `Updates the ${featureName} resource from your local backend.`,
       },
+      {
+        name: 'console',
+        description: `Opens the web console for the  ${featureName} category`,
```
Comment (SwaySway) [inline packages/amplify-category-auth/commands/auth.js]: nit: two spaces between the `${featureName}`

File: packages/amplify-category-hosting/commands/hosting.js [hunk 5]
```diff
@@ -30,6 +30,10 @@ module.exports = {
         name: 'remove',
         description: `Removes ${featureName} resource from your local backend which would be removed from the cloud on the next push command`,
       },
+      {
+        name: 'console',
+        description: `Opens the web console for the  ${featureName} category`,
```
Comment (SwaySway) [inline packages/amplify-category-hosting/commands/hosting.js]: nit: two spaces between the `${featureName}`

File: packages/amplify-category-api/commands/api.js [hunk 6]
```diff
@@ -37,6 +37,10 @@ module.exports = {
         name: 'add-graphql-datasource',
         description: 'Provisions the AppSync resources and its dependencies for the provided Aurora Serverless data source',
       },
+      {
+        name: 'console',
+        description: 'Opens the web console for selected api service',
```
Comment (SwaySway) [inline packages/amplify-category-api/commands/api.js]: nit: wording
`Opens the web console for a selected api service`

File: packages/amplify-category-xr/commands/xr.js [hunk 7]
```diff
@@ -23,6 +23,10 @@ module.exports = {
         name: 'remove',
         description: 'Removes XR resource from your local backend which would be removed from the cloud on the next push command',
       },
+      {
+        name: 'console',
+        description: `Opens the web console for the  ${featureName} category`,
```
Comment (SwaySway) [inline packages/amplify-category-xr/commands/xr.js]: nit: two spaces between the ${featureName}

File: packages/amplify-provider-awscloudformation/commands/awscloudformation.js [hunk 8]
```diff
@@ -10,5 +12,30 @@ module.exports = {
         context.print.error('Command not found');
       }
     }
+
+    const header = `amplify ${featureName} <subcommand>`;
+
+    const commands = [
+      {
+        name: 'configure',
+        description: `Configures the aws access credentials for the project.`,
```
Comment (SwaySway) [inline packages/amplify-provider-awscloudformation/commands/awscloudformation.js]: nit: upcase AWS

File: packages/amplify-cli/src/extensions/amplify-helpers/show-all-help.js [hunk 9]
```diff
@@ -68,6 +68,10 @@ function showAllHelp(context) {
       name: 'env',
       description: 'Displays and manages environment related information for your Amplify project.',
     },
+    {
+      name: 'console',
+      description: 'Opens the web console for selected cloud resource.',
```
Comment (SwaySway) [inline packages/amplify-cli/src/extensions/amplify-helpers/show-all-help.js]: nit: wording
`Opens the web console for the selected cloud resource.`

**Pull request merged**